var/spack/repos/builtin/packages/py-gcovr/package.py

```diff
@@ -26,3 +28,6 @@ class PyGcovr(PythonPackage):
     depends_on("py-jinja2", type=("build", "run"))
     depends_on("py-lxml", type=("build", "run"))
     depends_on("py-pygments", when="@5:", type=("build", "run"))
+    depends_on("py-pygments@2.13.0:", when="@7.2:", type=("build", "run"))
+    depends_on("py-colorlog", when="@7.2:", type=("build", "run"))
+    depends_on("py-tomli", when="@7.2:", type=("build", "run"))
```

Comment:
This needs to be more constrained depending on the `python` based on `setup.py`:

```
    install_requires=[
        "jinja2",
        "lxml",
        "colorlog",
        "pygments>=2.13.0",
        "tomli >= 1.1.0 ; python_version < '3.11'",
    ],
```

Reply:
How do you express `"tomli >= 1.1.0 ; python_version < '3.11'"` in Spack?

Reply:
Thanks, I committed your suggestion.